exercises/protein-translation/protein_translation_test.go

```diff
@@ -42,20 +116,11 @@ var proteinTestCases = []rnaCase{
 	{"UGGUGUUAUUAAUGGUUU", []string{"Tryptophan", "Cysteine", "Tyrosine"}},
 }
 
-func TestCodon(t *testing.T) {
-	for _, test := range codonTestCases {
-		actual := FromCodon(test.input)
-		if actual != test.expected {
-			t.Errorf("Protein Translation test [%s], expected [%s], actual [%s]", test.input, test.expected, actual)
-		}
-	}
-}
-
 func TestProtein(t *testing.T) {
 	for _, test := range proteinTestCases {
 		actual := FromRNA(test.input)
 		if !reflect.DeepEqual(actual, test.expected) {
-			t.Errorf("Protein Translation test [%s], expected %q, actual %q", test.input, test.expected, actual)
+			t.Errorf("FAIL: RNA Translation test: %s\nExpected: %q\nActual %q", test.input, test.expected, actual)
 		}
```

Comment:
Perhaps
```
+    t.Logf("PASS: %s", test.input)
```
here as well, to make the 'go test -v' have similar output for TestProtein, as TestCodon does.